roles/ansible_service_broker/tasks/generate_certs.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: Create service catalog cert directory
+  file:
+    path: "{{ openshift.common.config_base }}/ansible-service-broker"
+    state: directory
+    mode: 0755
+  check_mode: no
+
+- set_fact:
+    generated_certs_dir: "{{ openshift.common.config_base }}/ansible-service-broker"
+
+- name: Create self signing ca cert
+  command: 'openssl req -nodes -x509 -newkey rsa:4096 -keyout {{ generated_certs_dir }}/key.pem -out {{ generated_certs_dir }}/cert.pem -days 365 -subj "/CN=asb-etcd.ansible-service-broker.svc"'
```

Comment:
The namespace will probably need to be `openshift-ansible-service-broker`

Reply:
good catch, didn't notice the namespace in there.